ui/admin/tests/acceptance/accounts/change-password-test.js

```diff
@@ -77,9 +77,7 @@ module('Acceptance | accounts | change password', function (hooks) {
 
   test('can change password for account', async function (assert) {
     assert.expect(2);
-    // TODO: Failing due to a11y violation while in dark mode.
```

Comment:
Can we still keep the explanation and the ticket? I'm not a fan of having to manually going to jira to find context especially if they're not links

Reply:
Not a problem.